core/src/main/resources/db/cgds.sql

```diff
@@ -724,8 +727,34 @@ CREATE TABLE `clinical_event_data` (
   FOREIGN KEY (`CLINICAL_EVENT_ID`) REFERENCES `clinical_event` (`CLINICAL_EVENT_ID`) ON DELETE CASCADE
 );
 
+CREATE TABLE `gene_panel` (
+    `INTERNAL_ID` int(11) NOT NULL auto_increment,
+    `STABLE_ID` varchar(255) NOT NULL,
+    `DESCRIPTION` mediumtext,
+    PRIMARY KEY (`INTERNAL_ID`),
+    UNIQUE (`STABLE_ID`)
+);
+
+CREATE TABLE `gene_panel_list` (
+    `INTERNAL_ID` int(11) NOT NULL,
+    `GENE_ID` int(255) NOT NULL,
+    PRIMARY KEY (`INTERNAL_ID`, `GENE_ID`),
+    FOREIGN KEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE,
+    FOREIGN KEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
+);
+
+CREATE TABLE `gene_panel_profile_map` (
+    `SAMPLE_ID` int(11) NOT NULL,
+    `PROFILE_ID` int(11) NOT NULL,
+    `PANEL_ID` int(11) NOT NULL,
+    PRIMARY KEY (`SAMPLE_ID`, `PROFILE_ID`),
```

Comment:
@sheridancbio you understanding is correct. 

And yes, filling `sample_profile` with all samples of a profile would be a potential solution for #494. However, this does not remove the need for case lists as they could serve another use case (e.g. custom case lists). You should also be careful to make sure the other parts of the code, that use `sample_profile` in its current form, keep working as expected.